satpy/writers/cf_writer.py

```diff
@@ -1182,6 +325,47 @@ def save_datasets(self, datasets, filename=None, groups=None, header_attrs=None,
             written.append(res)
         return written
 
+    @staticmethod
+    def da2cf(dataarray, epoch=EPOCH, flatten_attrs=False, exclude_attrs=None,
+              include_orig_name=True, numeric_name_prefix='CHANNEL_'):
+        """Convert the dataarray to something cf-compatible.
+
+        Args:
+            dataarray (xr.DataArray):
+                The data array to be converted
+            epoch (str):
+                Reference time for encoding of time coordinates
+            flatten_attrs (bool):
+                If True, flatten dict-type attributes
+            exclude_attrs (list):
+                List of dataset attributes to be excluded
+            include_orig_name (bool):
+                Include the original dataset name in the netcdf variable attributes
+            numeric_name_prefix (str):
+                Prepend dataset name with this if starting with a digit
+        """
+        from satpy.writers.cf.dataarray import make_cf_dataarray
+        warnings.warn('CFWriter.da2cf is deprecated.'
+                      'Use satpy.writers.cf_writer.make_cf_dataarray instead.',
+                      DeprecationWarning, stacklevel=3)
+        return make_cf_dataarray(dataarray=dataarray,
+                                 epoch=epoch,
+                                 flatten_attrs=flatten_attrs,
+                                 exclude_attrs=exclude_attrs,
+                                 include_orig_name=include_orig_name,
+                                 numeric_name_prefix=numeric_name_prefix)
+
+    @staticmethod
+    def update_encoding(dataset, to_netcdf_kwargs):
+        """Update encoding info (deprecated)."""
+        from satpy.writers.cf.encoding import update_encoding
+
+        warnings.warn('CFWriter.update_encoding is deprecated. '
+                      'Use satpy.writers.cf.encoding.update_encoding instead.',
+                      DeprecationWarning, stacklevel=3)
```

Comment:
How long have these methods (this one and the one above it) been deprecated? Can we just remove them?